src/contents/Prove-Online-Facts.mdx

```diff
@@ -0,0 +1,298 @@
+
+TLSNotary addresses these challenges by introducing **Multi-Party Computation (MPC)** into the TLS communication process.
+
+# **How TLSNotary Works**
+
+The TLSNotary protocol operates through three main steps:
+
+1. **Multi-Party TLS Request:** Traditional TLS connections are between a user and a server, but TLSNotary introduces a third participant: the **Verifier**. Unlike a “[man-in-the-middle](https://en.wikipedia.org/wiki/Man-in-the-middle_attack),” the Verifier operates within a secure **Multi-Party Computation (MPC)** setup, ensuring that user data is not exposed in plaintext. To the server, the connection appears as a standard TLS request, preserving its integrity.*Alice and Bob can jointly manage the TLS connection, ensuring Bob can independently verify the data’s authenticity without seeing unshared portions.*
+2. **Selective Disclosure:** TLSNotary allows the Prover to **selectively disclose specific portions** of the data to the Verifier. Sensitive information can be redacted before sharing. ****Zero-Knowledge Proofs can be used to verify properties of redacted data without revealing the actual content.*Alice can choose which parts of the data to disclose, safeguarding her privacy.*
+3. **Data Verification:** The Verifier can then authenticate the disclosed data by validating the server certificate using trusted Certificate Authorities (CAs). This ensures that non-redacted portions of the data match their cryptographic proofs.*With this approach, Alice retains full control over her data while providing Bob with cryptographic proof of its authenticity.*
+
+## **TLSNotary and the Ethereum Foundation**
+
+TLSNotary is developed by the [**Privacy and Scaling Exploration (PSE)**](https://pse.dev/) research lab of the Ethereum Foundation. The PSE team is committed to conceptualizing and testing use cases for cryptographic primitives.
+
+# **Project: Prove Online Facts Without Revealing Too Much**
+
+**Overview:** This project demonstrates how to use **TLSNotary** to prove online facts, such as having a certain number of followers and living in a specific city, without exposing sensitive information like your exact profile details.
+
+**Use Case:** Ideal for scenarios like online deals or partnerships that require verification of certain criteria without compromising privacy.
+
+# **Setting Up the Development Environment**
+
+For this, we will go through **Three** **steps**. Most of these steps offer **two options** — choose one and move on to the next step. Often, the first option is easier from my perspective.
+
+## **1. Browser Extension**
+
+Install the TLSNotary extension in Chrome or Brave:
+
+***Option 1: Install via [Chrome Web Store](https://chromewebstore.google.com/detail/gcfkkledipjbgdbimfpijgbkhajiaaph)***
+
+Go to the Chrome Web Store and click **Add to Chrome**.
+
+![TLSNotary Chrome Extension](https://github.com/chiemezie1/app/blob/content-prove-online-facts/public/images/contents/TLSNotaryChromeExtension.png)
```

Comment:
Please use a direct relative link to your images. Also consider using an image description. Something like:

```
<div className="text-center">
  <img src="/images/contents/prove-online-facts/TLSNotaryChromeExtension.png" alt="TLSNotary Chrome Extension" />
  _TLSNotary Chrome Extension_
</div>
```